feature/kl_divergence.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+# Copyright FMR LLC <opensource@fidelity.com>
+# SPDX-License-Identifier: Apache-2.0
+
+from typing import NoReturn, Tuple
+
+import pandas as pd
+import numpy as np
+
+from scipy.special import rel_entr
+from feature.base import _BaseSupervisedSelector
+from feature.utils import Num, check_true
+
+
+class _KL_Divergence(_BaseSupervisedSelector):
+
+    def __init__(self, seed: int, num_features: Num, num_bins: Num = 100):
```

Comment:
agree keeping num_bins here.. not obvious to me how to make this part of Statistical without kwwargs which would require changing a lot of code.. this is good as-is